src/runtime/arc.ts

```diff
@@ -60,11 +63,33 @@ type DeserializeArcOptions = Readonly<{
   loader: Loader;
   fileName: string;
   context: Manifest;
-  inspectorFactory?: ArcInspectorFactory
+  inspectorFactory?: ArcInspectorFactory;
 }>;
 
 type SerializeContext = {handles: string, resources: string, interfaces: string, dataResources: Map<string, string>};
 
+export interface UnifiedStore {
```

Comment:
A comment here explaining what it's for would be great (and mentioning the fact that it's temporary -- it is supposed to be temporary, isn't it?)